internal/checker/cloudformation/rules/encryption/verify_rds_encryption_rest.rego

```diff
@@ -0,0 +1,52 @@
+# METADATA
+# title: "Verify the secure encryption status of data stored in the RDS at rest."
+# description: "This policy ensures enhanced security and compliance with data-at-rest encryption regulations for your cloud databases, providing robust protection and regulatory adherence."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html
+# custom:
+#   id: CB_CFT_007
+#   severity: HIGH
+package lib.cloudformation.CB_CFT_007
+import future.keywords.in
+
+resource := "AWS::RDS::DBInstance"
+
+is_valid {
+    some resources in input.Resources
+    resources.Type == resource
+}
+
+permitted_engines := {"aurora", "mysql", "oracle", "postgresql", "mariadb", "sqlserver"}
+
```

Comment:
remove line 29 here and just return resources since our goal is to return an object not an array
pass[resources]{
 //your logic
}